src/pybind/mgr/dashboard/controllers/host.py

```diff
@@ -304,13 +305,24 @@ class Host(RESTController):
                  },
                  responses={200: LIST_HOST_SCHEMA})
     @RESTController.MethodMap(version=APIVersion(1, 2))
-    def list(self, sources=None, facts=False):
+    def list(self, sources=None, facts=False, offset: int = 0, 
+             limit: int = 5, search: str = '', sort: str = ''):
         hosts = get_hosts(sources)
+        params = ['hostname']
+        paginator = ListPaginator(int(offset), int(limit), sort, search, hosts,
+                                  searchable_params=params, sortable_params=params,
+                                  default_sort='+hostname')
```

Comment:
I'm surprised that some of the brightest minds were burnt at the stake, but no one did anything about this type of indentation, which is awful and unmaintainable (lots of whitespacing, and if you refactor the name of the variable `paginator` or `ListPaginator`, you have to reindent all the lines).

Reply:
are you afraid of clicking two times to format that line (editors do that easily)? Even most of them format them on the fly.
I find this much easier to read.